analyzer/analyzer.py

```diff
@@ -635,9 +635,27 @@ def process_vdbench_data(self, path, dirname):
         result[dirname]["vdbench"] = output_vdbench_data
         return result
 
+    def get_lat_persent_dict(self,fio_str):
+        lat_percent_dict = {}
+        tmp_list = fio_str.split(',')
+        for i in tmp_list:
+            li = i.split('=')
+            while '' in li:li.remove('')
+            if len(li) == 2 and li[1] != '':
+                key = re.findall('.*?th',li[0].strip('\n').strip('| ').strip(' ').replace(' ',''),re.S)
+                value = re.match(r'\[(.*?)\]',li[1].strip('\n').strip(' ').replace(' ','')).groups()
+                if len(key) != 0 and len(value) != 0:
+                    lat_percent_dict[key[0]] = value[0]
+        return lat_percent_dict
+
     def process_fio_data(self, path, dirname):
         result = {}
         stdout, stderr = common.bash("grep \" *io=.*bw=.*iops=.*runt=.*\|^ *lat.*min=.*max=.*avg=.*stdev=.*\" "+path, True)
+        stdout1, stderr1 = common.bash("grep \" *1.00th.*],\| *30.00th.*],\| *70.00th.*],\| *99.00th.*],\| *99.99th.*]\" "+path, True)
+        lat_per_dict = {}
+        if stdout1 != '':
+            lat_per_dict = self.get_lat_persent_dict(stdout1)
```

Comment:
one thing need to add here, seems you only parse the data not the unit, if the unit is not msec, for example the unit is usec, then you need to use the function inside common.py to change the 299 usec to 0.299 msec.